specification/hybridcompute/resource-manager/Microsoft.HybridCompute/preview/2023-06-20-preview/examples/licenseProfile/LicenseProfile_CreateOrUpdate.json

```diff
@@ -0,0 +1,100 @@
+{
+  "parameters": {
+    "subscriptionId": "{subscriptionId}",
+    "resourceGroupName": "myResourceGroup",
+    "machineName": "myMachine",
+    "licenseProfileName": "default",
+    "api-version": "2023-06-20-preview",
+    "parameters": {
+      "location": "eastus2euap",
+      "properties": {
+        "esuProfile": {
+          "arcEsuEnrolled": "false",
+          "esuKeys": [
+            {
+              "esuSku": "esuSkuNumber1",
+              "licenseStatus": "licenseStatus1"
+            },
+            {
+              "esuSku": "esuSkuNumber2",
+              "licenseStatus": "licenseStatus2"
+            }
+          ]
+        }
+      }
+    }
+  },
+  "responses": {
+    "200": {
```

Comment:
the PUT/PATCH will always be an async response, as we need to send data to the HIS and confirm it was set. In the event that they try to turn it on for an ineligible server, we'll respond with a 400 and an error message

Reply:
ARM restriction